tools/RAiDER/delay.py

```diff
@@ -443,6 +463,17 @@ def build_cube(xpts, ypts, zpts, model_crs, pts_crs, interpolators):
     zmin = min(interpolators[0].grid[2])
     zmax = max(interpolators[0].grid[2])
 
+    # print("Output grid: ")
```

Comment:
Do these next few lines need to be deleted?

Reply:
There is no debugging mechanism for now. Will leave this in there if someone wants to test this out when adding new projection systems for new models